cli/command/image/build/context.go

```diff
@@ -44,7 +44,7 @@ func ValidateContextDirectory(srcPath string, excludes []string) error {
 	return filepath.Walk(contextRoot, func(filePath string, f os.FileInfo, err error) error {
 		if err != nil {
 			if os.IsPermission(err) {
-				return errors.Errorf("can't stat '%s'", filePath)
+				return errors.Errorf("can't stat: file not found or excluded by .dockerignore '%s'", filePath)
```

Comment:
This doesn't seem right. This is a permission error, not a not found error.
In addition, we should just wrap the real error `errors.Wrap(err, "error reading file")`. Generally the file name should already be in the error.